### ISSUE TITLE
Zurich

### ISSUE THREAD

**Issue by matthewberryman** [urbanriskmap/cognicity-reports#3]
As per
https://github.com/urbanriskmap/petabencana.id/issues/32
I would suggest we go straight to a scheduled lambda function rather than putting it in as part of reports, however. I guess for that, and for transforming qlue, etc. down the track, have them still as submodules of this reports master repo, and then this repo can package them (ala https://github.com/urbanriskmap/aws-lambda-image/blob/master/Makefile ) and document them? Leaving the powertrack as the odd one out.

**Comment by tomasholderness** [comment]
Agreed. I will start work on this soon -> NB. this is lower priority data source at the current time.
